src/main/scala/dotty/dokka/model/scalaModel.scala

```diff
@@ -61,3 +62,37 @@ case class HtmlContentNode(
   override def getChildren: JList[ContentNode] = Nil.asJava
   override def getExtra = extra
   override def withNewExtras(p: PropertyContainer[ContentNode]) = copy(extra = p)
+
+case class HackNestedTagWrapper(
+  name: String,
+  subname: String,
+  identTag: DocTag,
+  descTag: DocTag,
+)
+object HackNestedTagWrapper {
+  def encode(
+    name: String,
+    subname: String,
+    identTag: DocTag,
+    descTag: DocTag,
+  ) = CustomTagWrapper(P(List(identTag, descTag).asJava, Map.empty.asJava), s"$name#$subname")
+
+  def isNestedTagName(name: String) = name.contains("#")
+
+  def decodeName(name: String): Option[(String, String)] =
+    name.split("#", 2) match {
+      case Array(name, subname) => Some((name, subname))
+      case _ => None
+    }
+
+  def forceDecode(tag: NamedTagWrapper): HackNestedTagWrapper = {
+    val Array(name, subname) = tag.getName.split("#", 2)
+
+    HackNestedTagWrapper(
+      name,
+      subname,
+      tag.getChildren.get(0),
+      tag.getChildren.get(1),
+    )
+  }
+}
```

Comment:
There's hack that can be used to extend sealed classes from Kotlin in Scala: you need to pass `null` as parameter to constructor of sealed class e. g.:
We have Kotlin class like that:
```Kotlin
sealed class ExtraModifiers(val name: String)
```
We can extend it by doing:
```Scala
enum ScalaOnlyModifiers(val name: String, val prefix: Boolean) extends ExtraModifiers(name, null)
```
We already do this trick few times and it might be nicer to extend `NamedTagWrapper` than hack fields

Reply:
Right, I forgot that. I'll fix this code then.